msg_startup.go

```diff
@@ -105,3 +105,16 @@ func (m msg) CancelKeyData() (int32, int32, error) {
 	secret := int32(binary.BigEndian.Uint32(m[12:16]))
 	return pid, secret, nil
 }
+
+// parameterStatusMsg creates a new message providing parameter name and value
+func parameterStatusMsg(name, value string) msg {
+	length := /* TYPE+LEN */ 5 + len(name) + len(value) + /* TERMINATORS */ 2
```

Comment:
_Two terminators_
![Two terminators](http://cdn1.sciencefiction.com/wp-content/uploads/2015/06/terminator-arnold-and-emilia.jpg)

Reply:
:joy: